conf/distro/lmp-base.conf

```diff
@@ -0,0 +1,18 @@
+require conf/distro/include/lmp.inc
+
+DISTRO = "lmp-base"
```

Comment:
Can you please add 'DISTROOVERRIDES = "lmp:lmp-base"' here as well? That way the lmp-base distro will also use the lmp specific overrides.

Reply:
will do